### PR TITLE
Change UserProfile's num_seats to blank=True and null=True

### PR DESCRIPTION
**Pull request by dymontg**
Changes `num_seats` member of `UserProfile` object to accept null and blank values. Previously, it was only null values.

### Things Changed
- Added blank=True to `num_seats` in UserProfile model

### CODE REVIEW

File: auth/migrations/0003_auto_20170913_0740.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Generated by Django 1.11.4 on 2017-09-13 07:40
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('rest-auth', '0002_auto_20170830_0303'),
```
Comment (tapegun) [inline auth/migrations/0003_auto_20170913_0740.py]: what is purpose of second param?

File: auth/migrations/0003_auto_20170913_0740.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.4 on 2017-09-13 07:40
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```
Comment (tapegun) [inline auth/migrations/0003_auto_20170913_0740.py]: Why is it called migrations?

File: auth/migrations/0003_auto_20170913_0740.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+    dependencies = [
+        ('rest-auth', '0002_auto_20170830_0303'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='userprofile',
+            name='num_seats',
```
Comment (tapegun) [inline auth/migrations/0003_auto_20170913_0740.py]: I feel like there may be more variables we have missed here? I may be wrong though

File: auth/tests.py [hunk 4]
```diff
@@ -9,7 +9,7 @@
 
 from auth.views import UserProfileViewSet
 from auth.models import UserProfile
-
+from auth.serializers import UserProfileSerializer
 
 class UserProfileTests(APITestCase):
```
Comment (tapegun) [inline auth/tests.py]: is this class only supposed to be a test for our API?

File: auth/models.py [hunk 5]
```diff
@@ -10,7 +10,7 @@ class UserProfile(models.Model):
     # extra metadata
     dob = models.DateField(verbose_name='Date of Birth')
     car = models.BooleanField(verbose_name='Car?', default=False)
-    num_seats = models.PositiveSmallIntegerField(verbose_name='Number of Car seats', null=True)
+    num_seats = models.PositiveSmallIntegerField(verbose_name='Number of Car seats', null=True, blank=True)
```
Comment (tapegun) [inline auth/models.py:14]: if they don't enter a value isn't it automatically null? take out the opportunity for them to have a blank option, but this should work too, you will have to explain it to me though, it seems a bit redundant.

File: auth/tests.py [hunk 6]
```diff
@@ -35,11 +35,11 @@ def setUp(cls):
         cls.client = APIClient()
 
         # Create a User and UserProfile object
-        _user_data = cls.get_user_profile_data().pop('user')
-        _user = DjangoUser(**_user_data)
-        _user.set_password(_user_data['password'])
-        _user.save()
-        cls.user_profile = UserProfile.objects.create(user=_user, dob='1995-01-01')
+        serializer = UserProfileSerializer(data=cls.get_user_profile_data())
```
Comment (tapegun) [inline auth/tests.py]: what is the purpose of the serializer?

**Review by tapegun — APPROVED** [review]
comment your changes a bit better/ comment a bit better over all, because it is hard for me to understand without context and the commenting helps a bit with that

File: auth/models.py (hunk 5 above)
Comment (dymontg) [reply auth/models.py:14]: No, but good question. Check [this out](https://docs.djangoproject.com/en/dev/ref/models/fields/#field-options) for more info.

**Pull request merged**